src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -875,934 +881,9 @@ static bool processAreaScopeV2(const Scope* scoP, BSONObj &areaQuery)
 #define OPR_NOT_EXIST "NOT EXIST"
 
 
-
```

Comment:
Great to see all this stuff removed, now moved to the new classes :)

However, what about the #define above?

```
#define OPR_EXIST     "EXISTS"
#define OPR_NOT_EXIST "NOT EXIST"
```

Maybe the should be also removed?